tests/Spout/Writer/ODS/WriterWithStyleTest.php

```diff
@@ -268,30 +268,45 @@ public function testBorders()
         $this->writeToODSFileWithMultipleStyles($dataRows, $fileName, $styles);
 
         $styleElements = $this->getCellStyleElementsFromContentXmlFile($fileName);
+
         $this->assertEquals(3, count($styleElements), 'There should be 3 styles)');
 
+        // Use reflection for protected members here
+        $widthMapReflection = new \ReflectionProperty('Box\Spout\Writer\ODS\Helper\BorderHelper', 'widthMap');
```

Comment:
Use ReflectionHelper to retrieve the value: `$widthMap = ReflectionHelper::getStaticValue('Box\Spout\Writer\ODS\Helper\BorderHelper', 'widthMap');`